### DISCUSSION TITLE
Ingot for iBoss - Disables Force-Installed Extensions; Works on at least chromeOS 108

### DISCUSSION THREAD

**Discussion by BroJac5246** [3kh0/ext-remover#585] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
**This is for iBoss users ONLY**

While browsing this repository's commit history, I came across [this comment](https://github.com/3kh0/ext-remover/commit/cd086ebf7ee25fd61f5b0a94131b3dbdcf45c2e3#commitcomment-99406629). It links to [this repository](https://github.com/SpaceSaver/Ingot-for-iBoss), which is the Ingot extension disabler for iBoss (bookmarklet is located [here](https://raw.githubusercontent.com/SpaceSaver/Ingot-for-iBoss/main/bookmarklet.min.js)). I've been trying it out on chromeOS 108 and it works great! It can disable extensions that were force-installed by your organization, which I haven't seen any other exploits accomplish. It doesn't persist after a reboot, but it's super easy to re-run once bookmarked, and, once re-run, it'll automatically disable extensions that you previously disabled. Run it on this URL:
```
chrome-extension://kmffehbidlalibfeklaefnckpidbodff/restricted.html?re=1&bc=%3Cstyle%3Ediv%20{background:%20white%20!important;}%20p%20{color:%20rgba(0,0,0,0);}%20a%20{color:%20rgba(0,%200,%200,%201);}.left-content%20{display:%20none%20!important;}img%20{display:%20none%20!important;}form%20{display:%20none%20!important;}%20h2%20{display:%20none;}h3%20{display:%20none;}%3C/style%3E%3Ca+href=%22about:blank%22%20rel=%22opener%22%20target=%22_blank%22%3EClick%20here,%20then%20run%20a%20bookmarklet%20on%20the%20new%20tab.%3C/a%3E
```

Note: If you want to disable iBoss itself, then use the "Proxy Enabled" switch so you can still use the exploit, which relies on iBoss. If you run the code after a reboot, then previously-disabled extensions will be re-disabled. THERE IS AN EXCEPTION FOR IBOSS so that you can still use the exploit if you accidentally disabled it, but it does, of course, require a reboot to do so.

**Comment by GlizxyBoi** [comment]
> While browsing this repository's commit history, I came across [this comment](https://github.com/3kh0/ext-remover/commit/cd086ebf7ee25fd61f5b0a94131b3dbdcf45c2e3#commitcomment-99406629). It links to [this repository](https://github.com/SpaceSaver/Ingot-for-iBoss), which is the Ingot extension disabler for iBoss (bookmarklet is located [here](https://raw.githubusercontent.com/SpaceSaver/Ingot-for-iBoss/main/bookmarklet.min.js)). I've been trying it out on chromeOS 108 and it works great! It can disable extensions that were force-installed by your organization, which I haven't seen any other exploits accomplish. It doesn't persist after a reboot, but it's super easy to re-run once bookmarked, and, once re-run, it'll automatically disable extensions that you previously disabled. Run it on this URL:
> 
> ```
> chrome-extension://kmffehbidlalibfeklaefnckpidbodff/restricted.html?re=1&bc=%3Cstyle%3Ediv%20{background:%20white%20!important;}%20p%20{color:%20rgba(0,0,0,0);}%20a%20{color:%20rgba(0,%200,%200,%201);}.left-content%20{display:%20none%20!important;}img%20{display:%20none%20!important;}form%20{display:%20none%20!important;}%20h2%20{display:%20none;}h3%20{display:%20none;}%3C/style%3E%3Ca+href=%22about:blank%22%20rel=%22opener%22%20target=%22_blank%22%3EClick%20here,%20then%20run%20a%20bookmarklet%20on%20the%20new%20tab.%3C/a%3E
> ```
> 
> Note: If you want to disable iBoss itself, then use the "Proxy Enabled" switch so you can still use the exploit, which relies on iBoss. If you run the code after a reboot, then previously-disabled extensions will be re-disabled. THERE IS AN EXCEPTION FOR iBoss so that you can still use the exploit if you accidentally disabled it, but it does, of course, require a reboot to do so.

[Screen recording 2023-03-24 3.03.32 PM.webm](https://user-images.githubusercontent.com/94561504/227628037-52667334-19d8-4e09-9b7d-5ea288908b01.webm)

**Reply by Aka-but-nice** [reply]
You need iBoss.

**Reply by GlizxyBoi** [reply]
> You need iBoss.

ah ok
